src/main/resources/com/mathworks/ci/MatlabBuilder/RunTestsAutomaticallyOption/help-taPDFReportChkBx.html

```diff
@@ -1,10 +1,10 @@
 <div>
 <br>
 Generate a test report in PDF format, <b>testreport.pdf</b>, and save it in the <b>matlabTestArtifacts</b> folder of the Jenkins workspace.
-<br>
-<br>
-Note: Due to Jenkins Content Security Policy rules, the generated report might not open properly from within the Jenkins workspace. 
+<br><br>
+Due to Jenkins Content Security Policy rules, the generated report might not open properly from within the Jenkins workspace. 
```

Comment:
Is this true for pdf reports? I know it is for html

Reply:
Yes, this is true for PDF reports as well. The generated report will not load from within the jenkins workspace and this is due to jenkins content security policy - https://issues.jenkins-ci.org/browse/JENKINS-33195

In case of HTML, one can see the report but the scalar vector graphics (.svg) images displays weirdly - https://issues.jenkins-ci.org/browse/JENKINS-59276